test/node-api/test_buffer/test_buffer.c

```diff
@@ -123,6 +124,24 @@ static napi_value invalidObjectAsBuffer(napi_env env, napi_callback_info info) {
   return notTheBuffer;
 }
 
+static napi_value bufferFromArrayBuffer(napi_env env, napi_callback_info info) {
+  napi_status status;
+  napi_value arraybuffer;
+  void* data = NULL;
+  napi_value buffer;
+  size_t byte_length = 1024;
+
+  status = napi_create_arraybuffer(env, byte_length, &data, &arraybuffer);
+  NODE_API_ASSERT(env, status == napi_ok, "Failed to create arraybuffer");
+
+  status =
+      napi_create_external_buffer(env, byte_length, data, NULL, NULL, &buffer);
```

Comment:
I would expect that the test must be about the `node_api_create_buffer_from_arraybuffer`, and not the `napi_create_external_buffer`.